src/Speckle.Sdk.Dependencies/Serialization/ChannelSaver.cs

```diff
@@ -117,7 +103,7 @@ public async Task DoneSaving()
     }
   }
 
-  protected Exception? Exception { get; set; }
+  public Exception? Exception { get; set; }
```

Comment:
can we get away with a `protected set;`?

Reply:
can't put a protected thing in `IChannelSaver`  maybe internal

Reply:
can't do it :(